Assets/Scenes/Lighting.prefab

```diff
@@ -0,0 +1,271 @@
+%YAML 1.1
```

Comment:
Can you explain what this prefab is for? And see [this discussion](https://cvmlp.slack.com/archives/C05HWV5UZ1A/p1707154210737549?thread_ts=1707148506.826349&cid=C05HWV5UZ1A). I wonder if we can convert this to a piece of readable code.

Reply:
This shares the lights (directional lights) between the two scenes.

>I wonder if we can convert this to a piece of readable code.
The directional lights could be hard-coded. However, these are easier to reason about in the 3D view, and we may want to change them as they were roughly adjusted for the VR app.

Reply:
Ok. So this is literally lights? No attached script? Nothing here that is essential to the code running without error? Then this is the kind of thing I think we agree should be configured in the editor like you've done here. You might want to call it `DefaultLightSetup.prefab`, to convey that, if someone wants to deviate from these default lights, they should not use this fab and instead create something new.